store/postgres/src/relational_queries.rs

```diff
@@ -844,7 +844,7 @@ impl<'a> QueryFragment<Pg> for FilterQuery<'a> {
                 }
                 out.push_sql("select ");
                 out.push_bind_param::<Text, _>(&table.object)?;
-                out.push_sql(" as entity, e.id");
+                out.push_sql(" as entity, e.id, e.vid");
```

Comment:
Interesting, you're using the `vid` here since it potentially speeds up the join?

Reply:
Yes, exactly; it probably won't make a big difference, but should be a bit faster than joining on `block_range` and `id`.